tests/define_test_project.py

```diff
@@ -3,32 +3,32 @@
 from flow import FlowProject
 
 
-class TestProject(FlowProject):
+class ProjectTest(FlowProject):
```

Comment:
This class name is misleading, please rename to `_TestProject` and apply the same pattern wherever else this applies.